src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -29,3 +33,46 @@ def get(self, request, organization):
             raise ParseError(detail=six.text_type(err))
 
         return Response({"count": result["data"][0]["count"]})
+
+
+class OrganizationEventsRelatedIssuesEndpoint(OrganizationEventsEndpointBase, EnvironmentMixin):
+    def get(self, request, organization):
+        try:
+            params = self.get_filter_params(request, organization)
+        except OrganizationEventsError as e:
+            return Response({"detail": six.text_type(e)}, status=400)
+        except NoProjects:
+            return Response([])
+
+        possible_keys = ["transaction"]
+        lookup_keys = {key: request.query_params.get(key) for key in possible_keys}
+
+        if not any(lookup_keys.values()):
+            return Response(
+                {
+                    "detail": "Must provide one of {} in order to find related events".format(
+                        possible_keys
+                    )
+                },
+                status=400,
+            )
+
+        groups = []
+        try:
+            results = discover.get_related_issues(params=params, lookup_keys=lookup_keys)
+            group_ids = [r["group_id"] for r in results]
+            groups = Group.objects.filter(
+                id__in=group_ids,
+                project_id__in=params["project_id"],
+                status__in=[GroupStatus.UNRESOLVED, GroupStatus.RESOLVED, GroupStatus.IGNORED],
```

Comment:
Why resolved + ignored too?

Reply:
I thought it might be best to show everything? I removed resolved, but it might be good to show Ignored as well.

This is open to debate, I was trying to include more rather than less.

Reply:
Mm I don't want to hold this PR up over the statuses, so feel free to go with this :+1: but IMO if I ignore an issue I don't really want to see it again.

Reply:
:+1: to not showing ignored. Related issues should be actionable things that users can do. Ignored and resolved issues have had actions taken on them already.